xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
+  "type": "object",
+  "properties":{
+    "bt_piLast":{
+      "type": "string",
+      "description": "Last name of PI"
+    },
+    "bt_wavelength":{
```

Comment:
I assume that all these `"bt_something"` fields are supposed to be defined on a per beamtime level. Is wavelength on a per beamtime level?